changelog/1985.removal.rst

```diff
@@ -0,0 +1,17 @@
+Added the ``scheme`` parameter to ``HTTPConnection.set_tunnel`` to configure the scheme of the origin being tunnelled to.
```

Comment:
Should we mention that all those changes are there to distance ourselves from http.client a bit and allow future HTTP implementations in the future such as `h11`?

Reply:
Done in https://github.com/urllib3/urllib3/pull/2768/commits/afc660e87256768c13d0f0c0330b94bf6c0bb3c7